test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/InstrumentationPassIntegrationTest.cs

```diff
@@ -66,5 +58,28 @@ public void BasicTest()
             AssertCSharpDocumentMatchesBaseline(csharpDocument);
             Assert.Empty(csharpDocument.Diagnostics);
         }
+
+        private static TagHelperDescriptor CreateTagHelperDescriptor(
+            string tagName,
+            string typeName,
+            string assemblyName,
+            IEnumerable<Action<ITagHelperBoundAttributeDescriptorBuilder>> attributes = null)
```

Comment:
Nah, this method is just slightly more convenient for creating `TagHelperDescriptor`s. I personally prefer using the builder API's directly when possible.